London | 25-ITP-Sep | KME Blair | Sprint 1 | 07 Wireframe to Web Code

kohanman opened the pull request:
Self checklist

- [x] I have titled my PR with Region | Cohort | FirstName LastName | Sprint | Assignment Title
- [x] My changes meet the requirements of the task
- [x] I have tested my changes
- [x] My changes follow the [style guide](https://curriculum.codeyourfuture.io/guides/reviewing/style-guide/)

## Changelist

This pull request is for the wireframe webpage that has been modified from the basic template to a completed page with three articles and fulfils the accessibility requirements given by the Lighthouse analytic tool.

LonMcGregor commented on Wireframe/index.html:
```diff
@@ -8,25 +8,43 @@
+       A README is a file in the root folder of a repository, which displays under the files and folders in the root repo. 
+       This means a README is often the first item a visitor will see when visiting your repository in Github. README files typically include information on: what the project does; why the project is useful; how users can get started with the project; where users can get help with your project; and who maintains and contributes to the project.
+        </p>
+        <a href="">Read more</a>
```
Right now these links don't go anywhere. Can you think of where they should link to?

kohanman replied:
added html links to some further resources on the "read more" buttons

LonMcGregor commented on Wireframe/index.html:
```diff
@@ -8,25 +8,43 @@
   <body>
     <header>
-      <h1>Wireframe</h1>
+      <h1>Wireframes, README files and Git Branches </h1>
```
You updated the title here - great! is there anywhere else that it might be useful to change the title?

kohanman replied:
i will update the title in the head so it matches header

LonMcGregor commented on Wireframe/index.html:
```diff
@@ -8,25 +8,43 @@
     <footer>
       <p>
-        This is the default, provided code and no changes have been made yet.
+        This is the new, expertly provided code and evidence several changes have been made.
```
One of the acceptance criteria for this task is making sure the footer is fixed to the viewport. Do you know what that means? Can you fix that?

kohanman replied:
in the css

LonMcGregor reviewed (commented):
Good start on this sprint's tasks, I have spotted a few areas where you could improve code further

kohanman commented:
added background color on footer css to make sure the text is always readable and added some code to make the footer stretch across the bottom of the webpage because now it has a background it looks neater that way
